features/com.wso2telco.dep.hub.core.feature/src/main/resources/config/api-manager.xml

```diff
@@ -556,7 +556,6 @@
         <EnableQueryParamConditions>false</EnableQueryParamConditions>
     </ThrottlingConfigurations>
 
-    <!--MSISDN validation service endpoint - deployed via common gw/hub capps-->
-<msisdnValidationURL>http://localhost:8281/services/msisdnValidation</msisdnValidationURL>
```

Comment:
meke this url genaric as {http://esb-host}:{esb-nio-port}/services/msisdnValidation